lib/vanagon/platform/windows.rb

```diff
@@ -107,8 +109,9 @@ def generate_msi_package(project)
         dir_ref = "INSTALLDIR"
         # Actual array of commands to be written to the Makefile
         ["mkdir -p output/#{target_dir}",
-        "mkdir -p $(tempdir)/#{project.name}/staging",
```

Comment:
I think it appear here due to the rebasing I did when #270 was merged and the use of the ```"mkdir -p $(tempdir)/{staging,wix}"``` construct I have finally used - will see if I can make it clearer.